File: WebAPI/BritecWebAPI/BritecWebAPI/Models/ControleAplicacao/AplicacaoMassa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BritecWebAPI.Models.ControleAplicacao
{
    public class AplicacaoMassa : ParansInfoUsuario
    {
        public Nullable<long> id { get; set; }
        public string Nota { get; set; }
        public long Fase_id { get; set; }
        public double Estaca { get; set; }
        public DateTime data { get; set; }
        public DateTime HoraInicio { get; set; }
        public DateTime HoraFim { get; set; }
        public long Motorista_id { get; set; }
        public long Veiculo_id { get; set; }
        public double Largura { get; set; }
        public double Comprimento { get; set; }
        public double Toneladas { get; set; }
        public double Temperatura { get; set; }
        public double Espessura { get; set; }
    }
}

File: App/Coletor/app/models/AplicacaoMassa.js
exports.definition = {
	config: {
		columns: {
		    "usuarioId": "string",
		    "id": "int PRIMARY_KEY",
		    "Nota": "string",
		    "Fase_id": "int",
		    "Estaca": "int",
		    "data": "datetime",
		    "HoraInicio": "datetime",
		    "HoraFim": "datetime",
		    "Motorista_id": "int",
		    "Veiculo_id": "int",
		    "Largura": "double",
		    "Comprimento": "double",
		    "Toneladas": "double",
		    "Temperatura": "double",
		    "Espessura": "double"
		},
		adapter: {
			type: "sql",
			collection_name: "AplicacaoMassa",
			idAttribute: "id"
		}
	},
	extendModel: function(Model) {
		_.extend(Model.prototype, {
			// extended functions and properties go here
		});

		return Model;
	},
	extendCollection: function(Collection) {
		_.extend(Collection.prototype, {
			// extended functions and properties go here

			// For Backbone v1.1.2, uncomment the following to override the
			// fetch method to account for a breaking change in Backbone.
			/*
			fetch: function(options) {
				options = options ? _.clone(options) : {};
				options.reset = true;
				return Backbone.Collection.prototype.fetch.call(this, options);
			}
			*/
		});

		return Collection;
	}
};